src/System.Collections/src/System.Collections.builds

```diff
@@ -4,21 +4,8 @@
   <ItemGroup>
     <Project Include="System.Collections.csproj" />
     <Project Include="System.Collections.csproj">
-      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
I still think we should remove the live builds of these. When I chatted with @ericstj in the past we decided that if we ever needed to service one of these then we would do it from the release branch and then update the stable package we consume here to get that same fix into the latest package.